app/controllers/providers/used_delegated_functions_controller.rb

```diff
@@ -17,10 +17,26 @@ def update
     def save_continue_or_draft_and_update_scope_limitations
       return false unless save_continue_or_draft(@form)
```

Comment:
you could probably remove false here as without it it would be nil (still falsey)

Reply:
I think I prefer it be left in for clarity

Reply:
👍